pandas/core/frame.py

```diff
@@ -5717,6 +5717,14 @@ class    max    type
         monkey         mammal    NaN    jump
         """
         inplace = validate_bool_kwarg(inplace, "inplace")
+        if inplace:
```

Comment:
we need to replace the argument itself with `None` no? e.g. if False is passed this is still a problem